client/ayon_core/tools/publisher/widgets/widgets.py

```diff
@@ -1615,6 +1638,23 @@ def _input_value_changed(self, value, attr_id):
         if not instances or not attr_def or not plugin_name:
             return
 
+        # Update the styling of the associated label if is default or override
+        label_widget = self._attr_def_id_to_label.get(attr_id)
+        if label_widget:
+            # NOTE: The default returned by attr_def.default is returning a
+            # float, when it should be a int. So lets cast the widget value
+            # (argument to this method) to a float to match the type.
+            _value = value
+            if isinstance(_value, int):
+                _value = float(_value)
+            is_default = _value == attr_def.default
+            font = label_widget.font()
+            if is_default:
+                label_widget.setStyleSheet("")
+            else:
+                label_widget.setStyleSheet("font: bold")
+            label_widget.setFont(font)
```

Comment:
I wonder if we'd be better off setting a global additional stylesheet to the publisher UI that uses properties to set the fonts bold.

E.g. this could become
```css
*[nondefault=1] { font: bold }
```

Then I believe it's a matter on value change on the widget to do:
```python
widget.setProperty("nondefault", value != default)
```

See the example [here](https://doc.qt.io/qt-6/stylesheet-examples.html#customizing-using-dynamic-properties)

Reply:
Where is the stylesheets for the relevant widget in AYON?

Reply:
I think it's here https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/style/style.css#L891

Reply:
Yup, sounds like you're hitting the limitation I mentioned before where I linked [this](https://doc.qt.io/qt-5/stylesheet-syntax.html#selector-types).

See also bottom [here](https://wiki.qt.io/Dynamic_Properties_and_Stylesheets).

I guess maybe if we need to force the retrigger anyway we're just as well of just setting the stylesheets on value change or whatever.

Reply:
I changed it back to setting the stylesheet directly.